frontend/projects/valtimo/plugin-management/src/lib/components/plugin-external-permissions/plugin-external-permissions.component.scss

```
@@ -44,3 +44,13 @@
   display: block;
   margin-block-start: var(--cds-spacing-05);
 }
+
+:host ::ng-deep {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)